GraphClient.go

```diff
@@ -266,6 +271,30 @@ func (g *GraphClient) ListGroups(opts ...ListQueryOption) (Groups, error) {
 	return marsh.Groups, err
 }
 
+// ListUserGroups returns a list of all group ids the user is a member of
+// You can specify the securityGroupsEnabeled parameter to only return security group ids
+// Reference: https://docs.microsoft.com/en-us/graph/api/directoryobject-getmembergroups?view=graph-rest-1.0&tabs=http
```

Comment:
Please add an empty line before `// Refrence: https://...` so that in the godoc this line begins with a new line.